### DISCUSSION TITLE
Picture-in-picture on ios

### DISCUSSION THREAD

**Discussion by Jerrys-modz** [DonutWare/Fladder#142] [Ideas]
### Summary

Hi is there any plan to add PIP?

thank you for this awesome app!

### Basic Example

Picture in Picture 

### Drawbacks

I don’t think there is any drawbacks

**Comment by Ragulsundaram** [comment]
Honestly, I would kill for this feature.

**Comment by PartyDonut** [comment]
Let's continue this discussion in #348
